src/gc/gcinterface.h

```diff
@@ -546,16 +546,16 @@ class IGCHeap {
     virtual HRESULT Initialize() = 0;
 
     // Returns whether nor this GC was promoted by the last GC.
-    virtual BOOL IsPromoted(Object* object) = 0;
+    virtual bool IsPromoted(Object* object) = 0;
 
     // Returns true if this pointer points into a GC heap, false otherwise.
-    virtual BOOL IsHeapPointer(void* object, BOOL small_heap_only = FALSE) = 0;
+    virtual bool IsHeapPointer(void* object, bool small_heap_only = false) = 0;
```

Comment:
I'm not sure why MSVC *isn't* issuing a warning - I'd expect it to since it did for a bunch of other similar conversions. I can see if I can track down and fix the callers.